### PR TITLE
Grill and gibber changes

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Vastly improves the game with better balance, QOL and immersion.
Gibbing xenos now actually produces xeno meat instead of human meat.

You can now aggressive grab mobs and slam them into a hot grill.

https://user-images.githubusercontent.com/7869430/233966599-854f48e8-ea32-40b5-bb32-4c330ed3aa42.mp4

Somewhat refactors gibber code. Its still kinda wew, but it was exceptionally ??? before.

## Why It's Good For The Game
More immersive kitchen experiences.

## Changelog

:cl:
add: You can now slam mobs onto a hot grill
fix: Xenos now actually produce xeno meat when put into a gibber
fix: The gibber now actually makes a sound when gibbing
/:cl:

### CODE REVIEW

**Comment by Naaanii** [comment]
But can we eat the Xeno meat?

**Comment by Lumipharon** [comment]
I uh, haven't tried eating it raw, but yeah you can cook it/turn it into xeno burgers

**Comment by yyzsong** [comment]
Can xenos slam humans or even eachother onto hot grills?

File: code/game/objects/machinery/kitchen/gibber.dm [hunk 1]
```diff
@@ -95,106 +95,96 @@
...
+			sourcenutriment = 26
+		else //whatever else
 			totalslabs = 1
-			sourcenutriment = C.nutrition / 30 // small animals don't have as much nutrition
+			sourcenutriment = 13
 
-		for(var/i=1 to totalslabs)
-			var/obj/item/reagent_containers/food/snacks/meat/newmeat = new
-			newmeat.name = "[sourcename]-[newmeat.name]"
+	for(var/i=1 to totalslabs)
```
[the remark this answers is not in the document]
Comment (Lumipharon) [reply code/game/objects/machinery/kitchen/gibber.dm:153]: Probably not anymore, is there a neater way of doing it?

**Comment by Lumipharon** [comment]
> Can xenos slam humans or even eachother onto hot grills?

Warriors should be able to, yes. Warriors can actually stuff humans into the gibber, they just can't turn it on.

**Review by TiviPlus — APPROVED** [review]
XAanders working on revamping a lot of this anyway

**Pull request merged**